### ISSUE TITLE
Training v2vnet got the wrong result

### ISSUE THREAD

**Issue by shenxiaowrj** [DerrickXuNu/OpenCOOD#93]
I have trained v2vnet for 60 epochs. However, when I test the last epoch results, I get that all IOU is 0.

The output of the result is in the below picture.
![862291d09de027ddeadb5accbc1de81](https://github.com/DerrickXuNu/OpenCOOD/assets/100119921/1c44071c-cc51-4eb7-a6b0-81c978cb50b3)

Here is some background information about my training.

First, about the environment, spconv is spconv-cu113 2.3.6, pytorch is 1.11.0+cu113.
The more detailed environment information is in the bellow picture.
![3c9ec5fc004e6769827d22ffd6d85d3](https://github.com/DerrickXuNu/OpenCOOD/assets/100119921/3ce7ae26-1e7e-4492-b531-66dc0ea4922c)

Next, about the training and inference code.
I use 
python opencood/tools/train.py --hypes_yaml opencood/hypes_yaml/point_pillar_v2vnet.yaml
for train.
I use 
python opencood/tools/inference.py --model_dir opencood/logs/point_pillar_v2vnet_2023_06_05_06_40_58/ --fusion_method intermediate
for test.

Last, the training loss can be seen in the below picture.
![7975395262f8b2c8e7816e2a949236e](https://github.com/DerrickXuNu/OpenCOOD/assets/100119921/ca8718d1-a48d-47e9-b154-7b91ac9d560e)

Thanks a lot for helping me to solve this problem.

**Comment by DerrickXuNu** [comment]
V2VNet is hard to train, sometime you need some luck to have good weight initialization you train to coverage. I suggest directly using the checkpoint that I provide as a start point.

**Comment by shenxiaowrj** [comment]
> 

OK, thanks!
